sim/druid/claw.go

```diff
@@ -0,0 +1,80 @@
+package druid
+
+import (
+	"time"
+
+	"github.com/wowsims/classic/sim/core"
+)
+
+func (druid *Druid) registerClawSpell() {
+	flatDamageBonus := map[int32]float64{
+		20: 27,
+		28: 39,
+		38: 57,
+		48: 88,
+		60: 115,
+	}[druid.Level]
```

Comment:
Should just be able to use the level 60 115

Reply:
Removed lower levels of Claw